examples/misc/linalg.py

```diff
@@ -24,7 +24,7 @@ def callback_shape(ctx, param, value):
 
     def callback_opts(ctx, param, value):
         if value is True:
-            return ('blocking', 'simd', 'openmp', {'blockinner': True})
+            return 'advanced'
```

Comment:
this is not doing the same blocking/simd as `master`, because `blockinner: True` is dropped. Why?

Reply:
I vaguely remember that we talked about this in the past, maybe I am wrong....hmmm.. or it was due to the fact that no blocking happening nevertheless? Anyway reinstated.